Formula/n/nauty.rb

```diff
@@ -5,6 +5,7 @@ class Nauty < Formula
   mirror "https://users.cecs.anu.edu.au/~bdm/nauty/nauty2_8_8.tar.gz"
   sha256 "159d2156810a6bb240410cd61eb641add85088d9f15c888cdaa37b8681f929ce"
   license "Apache-2.0"
+  revision 1
```

Comment:
I don't think we need to force this to update for everyone.
```suggestion
```

Reply:
Oh, is that what bumping revision does? Without it there will be a new bottle but users need to update manually?

Reply:
Yes, it's a way to force an update to people.